tests/support/qunit_api.ts

```diff
@@ -5,12 +5,21 @@
 /* eslint-disable */
 
 // global.VF == Vex.Flow and is available in all of the tests.
+// TODO: Remove this after we have cleaned up the VF.* prefixes in the tests.
```

Comment:
I know I merged this in the other PR, however I just realized that the `support` directory is for drop-in third party code.

I'd prefer this to remain in the `tests/` directory -- we can call it `tests/qunit.ts`.

Reply:
OK, will do.